lesson06/assignment/form-validation.js

```diff
@@ -0,0 +1,115 @@
+
+// This variable represents the form itself for validation
+const getData = document.querySelector('.needs-validation');
+
+
+// The variable that has been declared starts with an 'event listener', then a submit function 
+  getData.addEventListener('submit', function(event) {
+    
```

Comment:
Your validation code looks good, but the assignment requires you display an alert if the form input is invalid. Your form does not indicate which field needs correction. There are many ways to do this, Bootstrap has a short example on their site you can reference, but you could definitely modify what you have to do this as well. https://getbootstrap.com/docs/5.0/forms/validation/